installscripts/wizard/scenarios/support/jazz_gitlab.py

```diff
@@ -21,15 +21,15 @@ def add_gitlab_config_to_files(parameter_list):
     replace_tfvars('scm_pathext', '/', get_tfvars_file())
 
 
-def get_and_add_docker_gitlab_config(gitlab_docker_path):
+def get_and_add_docker_gitlab_config(gitlab_docker_path, parameter_cred_list=[]):
     """
         Launch a Dockerized Gitlab server.
     """
     os.chdir(gitlab_docker_path)
     print("Running docker launch script  for gitlab")
 
     subprocess.call([
-        'sg', 'docker', './launch_gitlab_docker.sh', '|', 'tee', '-a',
+        'sg', 'docker',  './launch_gitlab_docker.sh %s %s' %( str(parameter_cred_list[1]) , str(parameter_cred_list[0])), '|', 'tee', '-a',
```

Comment:
Can we use the same password that we generated for Jazz Admin here? That way the experience will be similar for jazz admin and other users who register with Jazz.

Reply:
Yes. It is the same password generated for Jazz Admin